pkg/storage/unified/sql/test/integration_test.go

```diff
@@ -279,6 +283,75 @@ func TestBackendPrepareList(t *testing.T) {
 		assert.Equal(t, int64(4), continueToken.StartOffset)
 	})
 }
+func TestClientServer(t *testing.T) {
+	ctx := context.Background()
+	dbstore := infraDB.InitTestDB(t)
+
+	cfg := setting.NewCfg()
+	cfg.GRPCServerAddress = "localhost:0"
+	cfg.GRPCServerNetwork = "tcp"
+
+	features := featuremgmt.WithFeatures(featuremgmt.FlagUnifiedStorage)
+
+	svc, err := sql.ProvideService(cfg, features, dbstore, nil)
+	require.NoError(t, err)
+	var client resource.ResourceStoreClient
+
+	// Test with an admin identity
+	clientCtx := identity.WithRequester(context.Background(), &identity.StaticRequester{
+		Namespace:      identity.NamespaceUser,
```

Comment:
This will need updated before it can merge!

https://github.com/grafana/grafana/pull/90567